network-api/networkapi/settings.py

```diff
@@ -242,6 +242,8 @@
     'whitenoise.middleware.WhiteNoiseMiddleware',
     'django.middleware.gzip.GZipMiddleware',
     'networkapi.utility.middleware.TargetDomainRedirectMiddleware',
+    'networkapi.utility.middleware.HTMLCleanUpMiddleware' if DEBUG else None,
```

Comment:
I don't want to add this to prod, as it'll slow things down, but a source prettifier is quite useful. I can spin this off as a separate PR if we prefer, @mmmavis @cadecairos @KalobTaulien

Reply:
I vote for keeping them separate!